bindata/scripts/enable-rdma.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+REDHAT_RELEASE_FILE="/etc/redhat-release"
+
+function kmod_isloaded {
+  if grep --quiet '\(^ib\|^rdma\)' <(chroot /host/ lsmod); then
+    echo "RDMA kernel modules loaded"
+    true
+  else
+    echo "RDMA kernel modules not loaded"
+    false
+  fi
+}
+
+function trigger_udev_event {
+  echo "Trigger udev event"
+  chroot /host/ modprobe -r mlx4_en && chroot /host/ modprobe mlx4_en
+  chroot /host/ modprobe -r mlx5_core && chroot /host/ modprobe mlx5_core
+}
+
+if [ ! -f "$REDHAT_RELEASE_FILE" ]; then
+  exit 1
+fi
+
+if ! grep --quiet 'mlx4_en' <(chroot /host/ lsmod) && ! grep --quiet 'mlx5_core' <(chroot /host/ lsmod); then
```

Comment:
Q: Are mlx drivers loaded by default? is there any extra config required?

Reply:
I think those drivers will be loaded automatically if there is any such device detected. @Mmduh-483 @moshe010  Please correct me if I am wrong.

Reply:
@pliurh, yes you are right. rdma-core provide the following udev rules https://github.com/linux-rdma/rdma-core/blob/master/kernel-boot/rdma-hw-modules.rules which should load all the drivers for the detected device